pkg/signver/hashivault/config_auth.go

```diff
@@ -0,0 +1,25 @@
+package hashivault
+
+import (
+	"k8s.io/utils/env"
+)
+
+func getVaultAuthRoleId() string {
+	return env.GetString("WERF_VAULT_AUTH_ROLE_ID", env.GetString("VAULT_ROLE_ID", ""))
+}
+
+func getVaultAuthSecretId() string {
+	return env.GetString("WERF_VAULT_AUTH_SECRET_ID", env.GetString("VAULT_SECRET_ID", ""))
+}
+
+func getVaultAuthRole() string {
+	return env.GetString("WERF_VAULT_AUTH_ROLE", "")
+}
+
+func getVaultAuthJwt() string {
+	return env.GetString("WERF_VAULT_AUTH_JWT", "")
+}
+
+func getVaultAuthNamespace() string {
```

Comment:
Let's rename it to `WERF_VAULT_AUTH_METHOD` (`jwt`, `github`, `approle`, ...) or `WERF_VAULT_AUTH_PATH` (`/auth/github/login`). This clarifies the auth endpoint behavior and avoids confusion with Vault namespaces (`VAULT_NAMESPACE`).